cmd/dump.go

```diff
@@ -79,7 +79,10 @@ func runDump(ctx *cli.Context) error {
 	log.Printf("Packing dump files...")
 	z, err := zip.Create(fileName)
 	if err != nil {
-		os.Remove(fileName)
+		osErr := os.Remove(fileName)
```

Comment:
Here's a dump question... Why is it deleting a file it can't create in the first place? :trollface: